Koans/AboutConstants.cs

```diff
@@ -0,0 +1,114 @@
+using DotNetCoreKoans.Engine;
+using Xunit;
+
+namespace DotNetCoreKoans.Koans
+{
+    public class AboutConstants : Koan
+    {
+        const string C = "nested";
+
+        [Step(1)]
+        public void ConstantsMustBeInitalizedAsDeclared()
```

Comment:
This is a good first step. I think the next step should demonstrate that a constant cannot be changed.

Reply:
How do you propose to demonstrate that? Changing a constant value is a compilation error?

Reply:
@NotMyself please see latest.